1.split_data/scripts/nbconverted/split_data.py

```diff
@@ -54,10 +55,13 @@
 # create pandas dataframe with all indexes and their respective labels
 index_data = []
 for index in test_indexes:
-    index_data.append({"label": "test", "index": index})
+    index_data.append({"labeled_data_index": index, "label": "test"})
 for index in train_indexes:
-    index_data.append({"label": "train", "index": index})
+    index_data.append({"labeled_data_index": index, "label": "train"})
 index_data = pd.DataFrame(index_data)
+# put indexes into sorted order
+index_data = index_data.sort_values(["labeled_data_index"])
+
```

Comment:
consider chaining line 62 and 63:

```python
# I think this works
index_data = (
    pd.DataFrame(index_data)
    .sort_values(["labeled_data_index"])
)
```

Chaining is very pandas

Reply:
(`dplyr` also chains really elegantly)

Reply:
Added in https://github.com/WayScience/phenotypic_profiling_model/pull/19/commits/2f50b408e3448251863eb5497b50cceeb66159c2